bigtable/beam/workload-generator/src/main/java/bigtable/WorkloadGenerator.java

```diff
@@ -94,5 +123,11 @@ public void processElement(PipelineOptions po) throws IOException {
     Integer getWorkloadRate();
 
     void setWorkloadRate(Integer workloadRate);
+
+    @Description("The duration for the workload to run in minutes.")
+    @Default.Integer(10)
+    Integer getWorkloadDuration();
```

Comment:
nit - I would rename these variables to `get/setWorkloadDurationMinutes` to be clear what the time unit is